### PR TITLE
Align dashboard header menu 

### PR DESCRIPTION
**Pull request by sssoz**
# Realign header 

https://github.com/DOAJ/doajPM/issues/3702

Just realigning the menu. This happened because the menu item's markup might have been copied from a source with a layout with a different purpose. Simple fix was to just use the right menu item class (`label`), so that it is the same as the other ones in the menu. 

Also put back the `dropdown` block-level class as it needs to always accompany its modifiers when they are used, see [the modifiers section in this primer on BEM](https://dev.to/cglikpo/css-bem-made-easy-40i3). 

## Categorisation

This PR...
- [ ] has scripts to run
- [ ] has migrations to run
- [ ] adds new infrastructure
- [ ] changes the CI pipeline
- [ ] affects the public site
- [ ] affects the editorial area
- [ ] affects the publisher area
- [ ] affects the monitoring

## Basic PR Checklist

Instructions for developers:
* For each checklist item, if it is N/A to your PR check the N/A box
* For each item that you have done and confirmed for yourself, check Developer box (including if you have checked the N/A box)

Instructions for reviewers:
* For each checklist item that has been confirmed by the Developer, check the Reviewer box if you agree
* For multiple reviewers, feel free to add your own checkbox with your github username next to it if that helps with review tracking

### Code Style

- No deprecated methods are used
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

- No magic strings/numbers - all strings are in `constants` or `messages` files
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- ES queries are wrapped in a Query object rather than inlined in the code
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- Where possible our common library functions have been used (e.g. dates manipulated via `dates`)
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- Cleaned up commented out code, etc
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

- Urls are constructed with `url_for` not hard-coded
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
### Testing

- Unit tests have been added/modified
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- Functional tests have been added/modified
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- Code has been run manually in development, and functional tests followed locally
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

- Have CSS/style changes been implemented?  If they are of a global scope (e.g. on base HTML elements) have the downstream impacts of the change in other areas of the system been considered?
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

### Documentation

- FeatureMap annotations have been added
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- Documentation updates - if needed - have been identified and prepared for inclusion into main documentation (e.g. added and highlighted/commented as appropriate to this PR)
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- Core model documentation has been added to if needed: https://docs.google.com/spreadsheets/d/1lun2S9vwGbyfy3WjIjgXBm05D-3wWDZ4bp8xiIYfImM/edit
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

- Events and consumers documentation has been added if needed: https://docs.google.com/spreadsheets/d/1oIeG5vg-blm2MZCE-7YhwulUlSz6TOUeY8jAftdP9JE/edit
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer
  
- The docs for this branch have been generated and pushed to the doc site (see docs/README.md for details)
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

### Release Readiness

- If needed, migration has been created and tested locally
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

- Release sheet has been created, and completed as far as is possible https://docs.google.com/spreadsheets/d/1Bqx23J1MwXzjrmAygbqlU3YHxN1Wf7zkkRv14eTVLZQ/edit
  - [ ] N/A
  - [ ] Developer
  - [ ] Reviewer

- There has been a recent merge up from `develop` (or other base branch).  List the dates of the merges up from develop below
  - [date of merge up]

## Testing

List the Functional Tests that must be run to confirm this feature

1. ...
2. ...

## Deployment

What deployment considerations are there? (delete any sections you don't need)

### Configuration changes

What configuration changes are included in this PR, and do we need to set specific values for production

### Scripts

What scripts need to be run from the PR (e.g. if this is a report generating feature), and when (once, regularly, etc).

### Migrations

What migrations need to be run to deploy this

### Monitoring

What additional monitoring is required of the application as a result of this feature

### New Infrastructure

What new infrastructure does this PR require (e.g. new services that need to run on the back-end).

### Continuous Integration

What CI changes are required for this

### CODE REVIEW

File: cms/data/nav.yml [hunk 1]
```diff
@@ -36,8 +36,8 @@ entries:
     entries:
       - label: About DOAJ
         route: doaj.about             # ~~->About:WebRoute~~
-      - label: DOAJ at 20
-        route: doaj.at_20             # ~~->At20:WebRoute~~
+#      - label: DOAJ at 20
+#        route: doaj.at_20             # ~~->At20:WebRoute~~
```
Comment (richard-jones) [inline cms/data/nav.yml:40]: These changes seem unrelated, and will take a page offline, is that the intention

File: portality/view/doaj.py [hunk 2]
```diff
@@ -551,9 +551,12 @@ def faq():
...
     return render_template("layouts/static_page.html", page_frag="/about/index.html")
 
-@blueprint.route("/at-20/")
-def at_20():
-    return render_template("layouts/static_page.html", page_frag="/about/at-20.html")
+
+# TODO: Uncomment when ready for public access  - S.E. 2023-03-09
+# @blueprint.route("/at-20/")
+# def at_20():
+#     return render_template("layouts/static_page.html", page_frag="/about/at-20.html")
```
Comment (richard-jones) [inline portality/view/doaj.py:558]: Takes the DOAJ at 20 page offline, can you confirm that this is intended

**Review by richard-jones — COMMENTED** [review]
This PR does two things: removes the DOAJ at 20 page, and straightens the header.

Ideally those would be separate PRs, but it's ok to put them together if that's what's intended.  Can you confirm?

**Comment by sssoz** [comment]
> This PR does two things: removes the DOAJ at 20 page, and straightens the header.
> 
> Ideally those would be separate PRs, but it's ok to put them together if that's what's intended. Can you confirm?

Hmm I don't think this was a recent commit — my PR only intended to add three commits: 

1. https://github.com/DOAJ/doaj/pull/2325/commits/c0783e9606b9a47dce4a1dbf311df34ff90e0289
2. https://github.com/DOAJ/doaj/pull/2325/commits/59af7331f31811bfff70a56d0e76aa302414430c
3. https://github.com/DOAJ/doaj/pull/2325/commits/6fd2af69d91ad78b160e5cb9c830da5cd1083696 

Not sure how it picked up the old ones!

File: portality/templates/layouts/dashboard_base.html [hunk 3]
```diff
@@ -26,27 +26,25 @@ <h2 class="sr-only">DOAJ Dashboard</h2>
...
+                  <span><span data-feather="list" aria-hidden="true"></span> Editor dashboard</span>
+                </a>
+            </li>
+            {% if current_user.has_role("admin") %}
+              <li class="flex-center">
+                <a href="{{ url_for('dashboard.top_todo') }}" class="label">
+                  <span><span data-feather="list" aria-hidden="true"></span> Admin dashboard</span>
+                </a>
+              </li>
+            {% endif %}
```
Comment (amdomanska) [inline portality/templates/layouts/dashboard_base.html:47]: Looks like one of the {% if %} blocks has {% endif %} missing.

**Review by amdomanska — CHANGES_REQUESTED** [review]

**Comment by amdomanska** [comment]
@sssoz Could you please also clean up the commits if the changes that @richard-jones picked up are not intended for this PR?

File: cms/data/nav.yml (hunk 1 above)
Comment (sssoz) [reply cms/data/nav.yml:40]: I'm confused because these are changes committed by @Steven-Eardley that have been pulled in from `master`... I re-fetched and pulled and pushed to this PR and this is still here.

**Review by amdomanska — APPROVED** [review]

**Review by Steven-Eardley — APPROVED** [review]
